src/engine/renderer/tr_image.cpp

```diff
@@ -2731,6 +2731,11 @@ static void R_CreateReflectionCubeImages()
 // *INDENT-OFF*
 static void R_CreateWhiteCubeImage()
 {
+	if ( r_fastsky.Get() )
```

Comment:
r_fastsky is not latched. And shouldn't be in my opinion... it is not an expensive thing to build

Reply:
`r_fastsky` is currently latched, but this images is like blackCubeImage, we may still want to generate it in all cases. For example it may make possible to use `_whiteCube` in shaders to compose skyboxes. For example one may add some clouds in front of some white or black cube.

So I removed this commit as well.